drive_hw_bridge:
  ros__parameters:
    front_left:
      min_velocity: -10.0
      max_velocity: 10.0
      max_torque: 25.0
    middle_left:
      min_velocity: -10.0
      max_velocity: 10.0
      max_torque: 25.0
    back_left:
      min_velocity: -10.0
      max_velocity: 10.0
      max_torque: 25.0
    front_right:
      min_velocity: -10.0
      max_velocity: 10.0
      max_torque: 25.0
    middle_right:
      min_velocity: -10.0
      max_velocity: 10.0
      max_torque: 25.0
    back_right:
      min_velocity: -10.0
      max_velocity: 10.0
      max_torque: 25.0
